GCC Version 4.8.5 does not support assignment and move semantics for streams

GCC Version 4.8.5 does not support assignment and move semantics for streams

## Current behavior
Build fails for Tshirt_C_Realization_Test.cpp because of above.

Tshirt_C_Realization_Test.cpp:130:35: error: use of deleted function ‘std::basic_ifstream<char>& std::basic_ifstream<char>::operator=(const std::basic_ifstream<char>&)’


## Expected behavior
Successful build.

PR to follow with work-around of stream opening data file.

--- a/test/realizations/catchments/Tshirt_C_Realization_Test.cpp
+++ b/test/realizations/catchments/Tshirt_C_Realization_Test.cpp
@@ -127,7 +127,7 @@ void Tshirt_C_Realization_Test::TearDown() {
 void Tshirt_C_Realization_Test::open_standalone_c_impl_data_stream() {
     ASSERT_FALSE(standalone_output_data_file.empty());
 
-    standalone_data_ingest_stream = std::ifstream(standalone_output_data_file.c_str());
+    standalone_data_ingest_stream.open(standalone_output_data_file.c_str());
 
     ASSERT_TRUE(standalone_data_ingest_stream.is_open());
 

